### PR TITLE
combine dns docs

### LINKED ISSUE #673: Revise domains articles to create one complete document
Update/combine the following docs to create ONE complete Domains doc:

- Domains: (background/conceptual doc) https://pantheon.io/docs/articles/sites/domains/
- Add Domain: https://pantheon.io/docs/articles/sites/domains/adding-a-domain-to-a-site-environment
- Configure DNS: https://pantheon.io/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/

Outline:

### PR DESCRIPTION
**Pull request by nataliejeremy**
This PR closes #673 and:
- Combines Configuring DNS and Adding a Domain docs into Domains.md so that all info is in one place
- Deletes /docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site and /docs/articles/sites/domains/adding-a-domain-to-a-site-environment.md
- Removes links to the two deleted docs
- Removes the word "tool" from Domains and SSL references

### CODE REVIEW

**Comment by bmackinney** [comment]
This is a good start. Comments coming in-line.

File: source/docs/articles/going-live.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ If your site plan is Business, Professional, or Personal **or** using **HTTP** a
 		<tr>
 			<td class="complete">[ ]</td>
 			<td class="action">Update DNS with Pantheon DNS record</td>
-			<td><a href="/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/">DNS records for directing your domain to your Pantheon site</a></td>
+			<td><a href="/docs/articles/sites/domains/">DNS records for directing your domain to your Pantheon site</a></td>
```
Comment (bmackinney) [inline source/docs/articles/going-live.md]: Add the anchor to the H2

File: source/docs/articles/sites/domains.md [hunk 2]
```diff
@@ -1,52 +1,97 @@
...
+Once site plans are changed to a paid service level, custom domains can be added to each environment. Securing visitor traffic with HTTPS communication requires uploading a custom certificate and setting the DNS `A` record to a dedicated IP. For all sites in all configurations, the recommended DNS records will be provided on the Domains and SSL tab.
 
-* [Adding a Domain to a Site Environment](/docs/articles/sites/domains/adding-a-domain-to-a-site-environment/)
-* [Enable SSL for Secure HTTPS Communication](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication/)
-* [DNS Records for Directing Your Domain to Your Pantheon Site](/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/)
+## Step 1: Decide Which Type of Domain You Want
+
+**Examples of HTTPS Sites**
+
+https://www.example.com
```
Comment (bmackinney) [inline source/docs/articles/sites/domains.md]: - [ ] Highlight as Recommended, 
- [ ] link to why 
- [ ] and/or possibly briefly explain.

File: source/docs/articles/sites/domains.md [hunk 3]
```diff
@@ -1,52 +1,97 @@
...
+
+https://example.com and https://sub.example.com
```
Comment (bmackinney) [inline source/docs/articles/sites/domains.md]: - [ ] split these up, 
- [ ] change all `sub.` to https://`subdomain.`example.com,  
- [ ] add `[https://example.com/subdirectory](/source/docs/articles/sites/domains/fastly-domain-masking/)` 
- [ ] (more accurate than current URL requires also moving that article into the domains directory, adding redirect to .io)

**Comment by rachelwhitton** [comment]
Linkchecker output:
```
Start checking at 2015-07-09 12:00:01-005

Statistics:
Downloaded: 6.23MB.
Content types: 390 image, 4555 text, 0 video, 0 audio, 1838 application, 0 mail and 10722 other.
URL lengths: min=15, max=165, avg=54.

That's it. 17505 links in 344 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2015-07-09 12:02:18-005 (2 minutes, 16 seconds)
```

@nataliejeremy - the description here states:
>Combines Configuring DNS and Adding a Domain docs into Domains.md so that all info is in one place
>Deletes /docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site and /docs/articles/sites/domains/adding-a-domain-to-a-site-environment.md

But I don't see that those have been deleted or combined. Is that still planned?

**Comment by nataliejeremy** [comment]
@rachelwhitton It's in that PR

**Comment by nataliejeremy** [comment]
@rachelwhitton Hmmm, weird...I deleted them in atom when I was working on the doc. Let me look into it.

**Comment by nataliejeremy** [comment]
@rachelwhitton Please add redirects for the following:
/docs/articles/sites/domains/adding-a-domain-to-a-site-environment.md to /docs/articles/sites/domains.md

/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site.md to /docs/articles/sites/domains.md

**Comment by rachelwhitton** [comment]
+1 - I can add the redirects after this has been merged and deployed.

**Comment by nataliejeremy** [comment]
@bmackinney can you take one last look?

File: source/docs/articles/sites/domains.md [hunk 4]
```diff
@@ -1,52 +1,106 @@
...
 * DNS `A` record (www-redirector) - `192.237.224.60`
 * DNS `AAAA` record (www-redirector) - `2001:4801:7901:0000:c5ce:526c:0000:001a`
+* [Redirect non-www to www in settings.php or wp-config.php](/docs/articles/sites/code/redirect-incoming-requests/#redirect-to-a-common-domain)
 
 _Subdomains_
 
 * DNS `CNAME` record - `env-sitename.pantheon.io`
 
-### DNS Records for Legacy GotPantheon.com Base Domains
+### Legacy DNS Configuration for Pantheon.io Sites
```
Comment (bmackinney) [inline source/docs/articles/sites/domains.md]: This needs to stay gotpantheon.com

**Pull request merged**